packages/tokens/dist/cloud-email/helpers/color.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Do not edit directly
+ * Generated on Thu, 29 Feb 2024 14:07:07 GMT
+ */
+
+.hds-border-primary { border: 1px solid var(--token-color-border-primary); }
```

Comment:
I think I spoke too soon.... it doesn't look like Foundation for Emails is honoring the `var()` functions that are used by the hds classes.

Uploading CleanShot 2024-03-04 at 17.33.28.mp4…